src/node_buffer.cc

```diff
@@ -219,7 +219,10 @@ MaybeLocal<Object> New(Isolate* isolate,
   CHECK(actual <= length);
 
   if (actual < length) {
-    data = static_cast<char*>(realloc(data, actual));
+    char* new_data = static_cast<char*>(realloc(data, actual));
```

Comment:
5 line bottom it will free if it will be nullptr but if we put a free data in here we will try to free same thing twice and we never can learn if it will be null or a value before that line executed